### PR TITLE
Updated blender

### PR DESCRIPTION
**Pull request by WestleyK**
This update includes:
 - fixed URL in README example map (it was missing `Rendering`)
 - Updated pull command
 - verifying the container
 - Include remote builder for blender build 
 - Removed def file from blender README

### CODE REVIEW

File: graphics/rendering/blender/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Blender is a free and open source software used for editing and rendering.
 
-In this example, we will install Blender in a [Ubuntu container](https://cloud.sylabs.io/library/library/default/ubuntu).
+In this example, we will run Blender in a [Ubuntu container 16.04](https://cloud.sylabs.io/library/library/default/ubuntu).
```
Comment (jchavezlavalle) [inline graphics/rendering/blender/README.md]: change this from run to "install and run" Blender

File: graphics/rendering/blender/README.md [hunk 2]
```diff
@@ -13,36 +13,42 @@ In this example, we will install Blender in a [Ubuntu container](https://cloud.s
...
 
-### To start, Make the working directory:
+## To start, Make the working directory:
 
 ```
 $ mkdir ~/blender
 $ cd ~/blender/
 ```
 
+We will run this example in the `~/blender/` directory, not necessary but its cleaner this way.
```
Comment (jchavezlavalle) [inline graphics/rendering/blender/README.md]: not necessary but **it's** cleaner this way.

File: graphics/rendering/blender/README.md [hunk 3]
```diff
@@ -83,86 +89,45 @@ $ singularity run --nv blender.sif my_scene.blend run/output 5
...
 
-The above examples are all using the –nv option, for bringing into the container the nVidia libraries from the host.
+The above examples are all using the `–-nv` option, for bringing into the container the nVidia libraries from the host.
 
 
 <br>
 <br>
 
-### Building the contianer from a recipe:
+## Building the contianer from a definition file:
```
Comment (jchavezlavalle) [inline graphics/rendering/blender/README.md]: typo in word "container"

File: graphics/rendering/blender/blender.def [hunk 4]
```diff
@@ -1,40 +1,39 @@
...
-    singularity run blender.sif [scene file] [output directory] <frame | start frame:end frame>
+singularity run blender.sif [scene file] [output directory] <frame | start frame:end frame>
 
-    Example:
-    Using `my_scene.blend`, render all frames, and output into `run/output`
+Example:
+Using `my_scene.blend`, render all frames, and output into `run/output`
 
-        singularity run blender.sif my_scene.blend run/output
+    singularity run blender.sif my_scene.blend run/output
```
Comment (jchavezlavalle) [inline graphics/rendering/blender/blender.def]: Same here, just for context add `$` before the command, check and add this for all the help section commands

File: graphics/rendering/blender/blender.def [hunk 5]
```diff
@@ -1,40 +1,39 @@
...
 
 %post
-    apt-get update
-    apt-get -y install blender blender-data
+apt-get update
+apt-get -y install blender blender-data
 
 %help
-    singularity run blender.sif [scene file] [output directory] <frame | start frame:end frame>
+singularity run blender.sif [scene file] [output directory] <frame | start frame:end frame>
```
Comment (jchavezlavalle) [inline graphics/rendering/blender/blender.def]: To let them know this is a command, might need to add `$` so.. `$ singularity run blender.sif ...`

File: graphics/rendering/blender/README.md [hunk 6]
```diff
@@ -13,36 +13,42 @@ In this example, we will install Blender in a [Ubuntu container](https://cloud.s
...
-Then, pull the container from the library:
+### Then, pull the container from the library:
 
 ```
-$ singularity pull library://sylabs/examples/blender.sif:latest
+$ singularity pull blender.sif library://sylabs/examples/blender:latest
 ```
 
-If your building the container from a recipe, click [here](#building-the-contianer-from-a-recipe) or scroll down.
+If your building the container from a definition file, click [here](#building-the-contianer-from-a-definition-file) or scroll down.
```
Comment (jchavezlavalle) [inline graphics/rendering/blender/README.md]: If **you're** ...

File: graphics/rendering/blender/README.md [hunk 7]
```diff
@@ -13,36 +13,42 @@ In this example, we will install Blender in a [Ubuntu container](https://cloud.s
...
 $ mkdir ~/blender
 $ cd ~/blender/
 ```
 
+We will run this example in the `~/blender/` directory, not necessary but its cleaner this way.
+
 <br>
 
-Then, pull the container from the library:
+### Then, pull the container from the library:
```
Comment (jchavezlavalle) [inline graphics/rendering/blender/README.md]: change title by "Pull the container from library"

**Review by jchavezlavalle — COMMENTED** [review]
left some comments in here to fix some typos @WestleyK :+1:

File: graphics/rendering/blender/blender.def (hunk 4 above)
Comment (WestleyK) [reply graphics/rendering/blender/blender.def]: :+1:

File: graphics/rendering/blender/README.md (hunk 1 above)
Comment (WestleyK) [reply graphics/rendering/blender/README.md]: :+1:

File: graphics/rendering/blender/README.md (hunk 2 above)
Comment (WestleyK) [reply graphics/rendering/blender/README.md]: :+1:

File: graphics/rendering/blender/README.md (hunk 3 above)
Comment (WestleyK) [reply graphics/rendering/blender/README.md]: :+1:

File: graphics/rendering/blender/blender.def (hunk 5 above)
Comment (WestleyK) [reply graphics/rendering/blender/blender.def]: :+1:

File: graphics/rendering/blender/README.md (hunk 6 above)
Comment (WestleyK) [reply graphics/rendering/blender/README.md]: :+1:

File: graphics/rendering/blender/README.md (hunk 7 above)
Comment (WestleyK) [reply graphics/rendering/blender/README.md]: :+1:

**Review by jmstover — APPROVED** [review]

**Review by jchavezlavalle — APPROVED** [review]
lgtm :+1: @WestleyK

**Review by jchavezlavalle — APPROVED** [review]

**Pull request merged**